lib/Wikia/src/Service/User/Preferences/PreferenceService.php

```diff
@@ -2,26 +2,243 @@
 
 namespace Wikia\Service\User\Preferences;
 
-use Wikia\Domain\User\GlobalPreference;
+use Doctrine\Common\Cache\CacheProvider;
+use Wikia\Domain\User\Preferences\LocalPreference;
+use Wikia\Domain\User\Preferences\UserPreferences;
+use Wikia\Logger\Loggable;
+use Wikia\Persistence\User\Preferences\PreferencePersistence;
+use Wikia\Util\WikiaProfiler;
 
-interface PreferenceService {
+class PreferenceService {
```

Comment:
The class that `PreferenceService` is replacing was not behind an interface, so I removed the interface here alone with its only implementation (`PreferenceKeyValueService`) since they seemed redundant. 

I guess I can add an interface here, though realistically the only consumer of this is the `User` object, which is untestable for much bigger reasons.